prometheus/prometheus_targets.yml

```diff
@@ -21,3 +21,5 @@
     - gitserver-0:6060
     - searcher-0:6060
     - symbols-0:6060
+    - postgres-exporter:9187
```

Comment:
This line should go below lsif-server on line 18; the area you put it are for entries that must be duplicated for multiple replicas